labs/lab1/chapter1.md

```diff
@@ -34,40 +34,40 @@ Now check on the status of docker.
 $ systemctl status docker
 ```
```

Comment:
we talk about podman in the words, but all of these commands are referencing docker